package.json

```diff
@@ -14,6 +14,7 @@
     "@headlessui/react": "^1.7.18",
     "clsx": "^1.2.1",
     "date-fns": "^2.30.0",
+    "graphql-request": "^6.1.0",
```

Comment:
We can make `graphql` calls without this package. Check how it's done in `ActiveProjects.tsx`